moon/vehicles/rover.py

```diff
@@ -182,7 +187,7 @@ def on_joint_effort(self, data):
 
         elif self.drive_radius == 0:
             # turning in place if radius is 0 but speed is non-zero
-            e = 40
```

Comment:
well, this PR was meant to be more like continuation of changes needed to make moon solution in master work again after the most recent NASA change rather than just adding sim time; however the PR is called that way as it's more informative than some generic 'post NASA change fixes' type of name and 95% of its content is about sim time